### PR TITLE
feat: Configurable levels

### PR DESCRIPTION
**Pull request by carstencodes**
At our company, we're using SARIF to verify the code style of our sources.

Some of our proto files do intentionally not follow some of the proto.dev styling rules.

In order to keep the results of the checks, but to mitigate we would like to set the linter to a lower level as notice. 

The severity is now part of a rule configuration, the result will be directly added to the sarif report.

### CODE REVIEW

File: internal/addon/plugin/shared/gRPCServer.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 
 // GRPCServer is the implementation of RuleSet.
 type GRPCServer struct {
+	proto.UnimplementedRuleSetServiceServer
```
Comment (yoheimuta) [inline internal/addon/plugin/shared/gRPCServer.go:11]: Does this line pertain to your feature?

**Review by yoheimuta — APPROVED** [review]
Thank you for your awesome contribution!
I have a question although I approved it for now.

**Comment by yoheimuta** [comment]
I have just released the latest update as version [v0.45.0](https://github.com/yoheimuta/protolint/releases/tag/v0.45.0).

**Comment by carstencodes** [comment]
Great thanks.

**Pull request merged**